db/db_iter.cc

```diff
@@ -1047,120 +1062,154 @@ bool DBIter::IsVisible(SequenceNumber sequence) {
   }
 }
 
-void DBIter::Seek(const Slice& target) {
-  PERF_CPU_TIMER_GUARD(iter_seek_cpu_nanos, env_);
-  StopWatch sw(env_, statistics_, DB_SEEK);
-  status_ = Status::OK();
-  ReleaseTempPinnedData();
```

Comment:
Why these few lines are moved to after TraceIteratorSeek?

Reply:
I just feel that they logically continuous to lines below, so I want to move them together. If you think it's not the case, I can move it back.